includes/class-wc-payments.php

```diff
@@ -1763,16 +1766,16 @@ public static function wcpay_show_old_woocommerce_for_hungary_sweden_and_czech_r
 
 		switch ( $store_currency ) {
 			case 'HUF':
-				/* translators: %1$s: The current WordPress version used by the store */
-				$notice = __( 'The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Hungarian business. Please update WooCommerce to version 7.8 or above (you are using %1$s). You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' );
+				/* translators: %1$s: WooPayments, %2$s: The current WordPress version used by the store */
+				$notice = __( 'The WooCommerce version you have installed is not compatible with %1$s for a Hungarian business. Please update WooCommerce to version 7.8 or above (you are using %2$s). You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' );
```

Comment:
Same comment as above for these next few lines re: moving `WooCommerce` out of the translation string and injecting it.

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/6581/commits/a10101f8f95ed321704be33f29bad8ac81c0aa01